modules/item/src/test/java/unnamed/mmo/crafting/TestCraftingRecipeIntegration.java

```diff
@@ -0,0 +1,56 @@
+package unnamed.mmo.crafting;
+
+import net.minestom.server.MinecraftServer;
+import net.minestom.server.item.ItemStack;
+import net.minestom.server.item.Material;
+import org.junit.jupiter.api.Test;
+import unnamed.mmo.item.crafting.CraftingInventory;
+import unnamed.mmo.item.crafting.RecipeList;
+import unnamed.mmo.item.crafting.ShapelessCraftingRecipe;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class TestCraftingRecipeIntegration {
```

Comment:
Not an integration test, also would like to see some more tests for particular recipe types, actually taking the items, etc